### ISSUE TITLE
Deviance

### ISSUE THREAD

**Issue by anujkhare** [anujkhare/iregnet#17]
Deviance is just the residual taken to the log-likelihood scale.

**Comment by tdhock** [comment]
I think the deviance is just the negative log-likelihood (aka the loss function), which you can use as a model selection criterion for the cv.iregnet function (minimize the deviance on a held-out validation set)
